src/sonic-py-common/sonic_py_common/interface.py

```diff
@@ -0,0 +1,43 @@
+"""
+ Dictionary of SONIC interface name prefixes. Each entry in the format
+ "Human readable interface string":"Sonic interface prefix"
+ Currently this is a static mapping, but in future could be stored as metadata in DB.
+"""
+
+SONIC_INTERFACE_PREFIXES = {
+  "Ethernet-FrontPanel":"Ethernet",
+  "PortChannel":"PortChannel",
+  "Vlan":"Vlan",
+  "Loopback":"Loopback",
+  "Ethernet-Backplane":"Ethernet-BP"
```

Comment:
Add spaces after colons to align with PEP8 standards.

Reply:
Thanks Joe ! updated the definition.